controllers/articles.controllers.js

```diff
@@ -8,4 +8,14 @@ exports.getArticleById = (request,response,next) => {
     .catch((err)=>{
         next(err)
     })
+}
+exports.getArticles = (request, response, next) => {
+    const  {sort_by} = request.query
+    const {order} = request.query
```

Comment:
You can do these two on one line, destructure from one request.query and list sort_by, order